samples/core/GetStarted/Model.cs

```diff
@@ -9,7 +9,7 @@ public class BloggingContext : DbContext
         public DbSet<Post> Posts { get; set; }
 
         protected override void OnConfiguring(DbContextOptionsBuilder options)
-            => options.UseSqlite("Data Source=blogging.db");
+            => options.UseSqlite(@"Data Source=C:\blogging.db");
```

Comment:
The VS Mac tab is available but probably not what you want. Need either code comments or note before code for mac/linix

Reply:
How does this note look?

Reply:
Can also add a comment here in code what to use if your are using mac/linux.

Reply:
I agree, it turns out even folks reading localized content prefer helpful english code comments. I hate the chrome of a note. I'd move this to the code comments and remove the note. Note chrome detracts from the flow of the article.

Reply:
Changed to note into a code comment